annotation/src/main/java/org/cbioportal/genome_nexus/annotation/domain/MutationAssessor.java

```diff
@@ -0,0 +1,70 @@
+package org.cbioportal.genome_nexus.annotation.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+// @Document(collection = "hotspots")
+ @JsonIgnoreProperties(ignoreUnknown = true)
+ @JsonInclude(JsonInclude.Include.NON_NULL)
+    public class MutationAssessor
+    {
+
+        // user-input variant
+        private String variant;
+
+        private String hugoSymbol;
+        private String functionalImpact;
+        private String fScore;
```

Comment:
I think this one should be a number, not a string. Also `functionalImpactScore` might be a better variable name than `fScore`.